payment/service.go

```diff
@@ -312,6 +316,47 @@ func (s *Service) CancelOrder(orderID uuid.UUID) error {
 	return s.Datastore.UpdateOrder(orderID, OrderStatusCanceled)
 }
 
+// SetOrderTrialDays set the order's free trial days
+func (s *Service) SetOrderTrialDays(ctx context.Context, orderID *uuid.UUID, days int64) error {
+	// get the order
+	err := s.Datastore.SetOrderTrialDays(ctx, orderID, days)
```

Comment:
might be simpler to just have this return updated order?

Reply:
addressed: https://github.com/brave-intl/bat-go/pull/945/commits/bc0dfc7a96b474900bad381e9553bed06c6491a1